package.json

```
@@ -292,7 +293,7 @@
 			"wp-scripts lint-style"
 		],
 		"*.{js,ts,tsx}": [
```

Should it cover YAML now?
```suggestion
		"*.{js,ts,tsx,yml}": [
```

---

Ah, missed that one! Thanks!